docs/appendices/migration-guides/index.mdx

```diff
@@ -14,11 +14,60 @@ import TabItem from "@theme/TabItem";
 
 ## `2.18.0` -> `2.19.0`
 
+### Allow report-only frames for Adyen
+
+You may had to allow all iframes with Adyen to load 3DS2 authorization iframes
```

Comment:
```suggestion
If you were using Adyen as a payment provider you might have allowed all iframes to load, to prevent blocking  3DS2 authorization iframes
```